NtupleProducer/test/check_dataset_availability.py

```diff
@@ -1,17 +1,24 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # coding: utf-8
 
 """
 Checks if all files of one or multiple datasets are available on non-blacklisted sites.
 """
 
 import re
+import sys
 import json
 import subprocess
+import urllib
+import urllib.request
+import json
 
 import tqdm
 
 
+# dynamic file listing all usable and blacklisted sites
+usable_sites_url = "https://cmssst.web.cern.ch/cmssst/analysis/usableSites.json"
```

Comment:
Isn't that used by default anyway whenever submitting a crab job?

Reply:
Exactly. That's why it might be necessary to use the exact same black list to check if samples will be reachable prior to submitting jobs. Otherwise, crab will issue a warning, but still continue submitting the rest, and basically consider the uncreachable files non-existing. I hope this doesn't apply to real data ...

Reply:
Every issue we've seen in the past few days affects data and MC the same unfortunately !

Reply:
I was about to write the same comment, data and MC are treated the same way :'(